### PR TITLE
Fix description for role property

### PR DESCRIPTION
**Pull request by pruthvi-ustepalle**
Adding to the description for the role property.

### CODE REVIEW

File: api-reference/beta/api/team-post-members.md [hunk 1]
```diff
@@ -14,6 +14,9 @@ Namespace: microsoft.graph
 
 Add a new [conversation member](../resources/conversationmember.md) to a [team](../resources/team.md).
 
+> [!NOTE] 
+> By default, everyone has a `member` role. This property only contains additional qualifiers when relevant - for example, if the member has `owner` privileges, then the roles property would contain `owner` as one of the values. Similarly, if the member is a guest, then the roles property would contain `guest` as one of the values. A basic member should not have any values specified in the roles property.
```
Comment (AkJo) [inline api-reference/beta/api/team-post-members.md:18]: The 'roles' property will be empty by default for all members. This property only con...

File: api-reference/v1.0/api/team-post-members.md [hunk 2]
```diff
@@ -12,6 +12,9 @@ Namespace: microsoft.graph
 
 Add a new [conversationMember](../resources/conversationmember.md) to a [team](../resources/team.md).
 
+> [!NOTE] 
+> By default, everyone has a `member` role. This property only contains additional qualifiers when relevant - for example, if the member has `owner` privileges, then the roles property would contain `owner` as one of the values. Similarly, if the member is a guest, then the roles property would contain `guest` as one of the values. A basic member should not have any values specified in the roles property.
```
Comment (AkJo) [inline api-reference/v1.0/api/team-post-members.md:16]: The 'roles' property will be empty by default for all members. This property only con...

File: api-reference/v1.0/resources/conversationmember.md [hunk 3]
```diff
@@ -39,7 +39,7 @@ See also [aadUserConversationMember](aaduserconversationmember.md).
 |:---------------|:--------|:----------|
 |id|String| Read-only. Unique ID of the user.|
 |displayName| string | The display name of the user. |
-|roles| string collection | The roles for that user. |
+|roles| string collection | The roles for that user. By default, everyone has a `member` role. This property only contains additional qualifiers when relevant - for example, if the member has `owner` privileges, then the roles property would contain `owner` as one of the values. Similarly, if the member is a guest, then the roles property would contain `guest` as one of the values. A basic member should not have any values specified in the roles property. |
```
Comment (AkJo) [inline api-reference/v1.0/resources/conversationmember.md:42]: The roles for that user. By default, everyone has a `member` role. This property only contains additional ....

File: api-reference/beta/resources/conversationmember.md [hunk 4]
```diff
@@ -41,7 +41,7 @@ Represents a user in a [team](team.md) or a [channel](channel.md) or a [chat](ch
 |:---------------|:--------|:----------|
 |id|String| Read-only. Unique ID of the user.|
 |displayName| string | The display name of the user. |
-|roles| string collection | The roles for that user. |
+|roles| string collection | The roles for that user. By default, everyone has a `member` role. This property only contains additional qualifiers when relevant - for example, if the member has `owner` privileges, then the roles property would contain `owner` as one of the values. Similarly, if the member is a guest, then the roles property would contain `guest` as one of the values. A basic member should not have any values specified in the roles property. |
```
Comment (AkJo) [inline api-reference/beta/resources/conversationmember.md:44]: The roles for that user. By default, everyone has a `member` role. This property only contains additional ....

File: api-reference/v1.0/resources/conversationmember.md [hunk 5]
```diff
@@ -39,7 +39,7 @@ See also [aadUserConversationMember](aaduserconversationmember.md).
 |:---------------|:--------|:----------|
 |id|String| Read-only. Unique ID of the user.|
 |displayName| string | The display name of the user. |
-|roles| string collection | The roles for that user. |
+|roles| string collection | The roles for that user. This property only contains additional qualifiers when relevant - for example, if the member has `owner` privileges, then the roles property would contain `owner` as one of the values. Similarly, if the member is a guest, then the roles property would contain `guest` as one of the values. A basic member should not have any values specified in the roles property. |
```
Comment (AkJo) [inline api-reference/v1.0/resources/conversationmember.md:42]: The content writer team should review this and the other strings for clarity.

**Review by AkJo — APPROVED** [review]

**Review by Lauragra — APPROVED** [review]

**Pull request merged**